app/assets/javascripts/code/services/_draw_session.js

```diff
@@ -95,3 +96,17 @@ export const drawSession = (
 };
 
 const calculateHeatLevel = (heat, value) => heat.getLevel(value);
+
+const createMeasurementMarker = (measurement, idx, heat, map, suffix) => {
+  var roundedValue = Math.round(measurement.value);
+  var level = calculateHeatLevel(heat, roundedValue);
+  if (level){
```

Comment:
I'd suggest an early return here. It makes more clear what happens when level is falsy and keeps code linear